PrivateCollections/Ethiopia/MarigetaDamdaEndalaw/MD001.xml

```diff
@@ -0,0 +1,201 @@
+                                        </item>
+                                        <item xml:id="q9" n="8">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="73r" to="82v"/>                                            
+                                        </item>
+                                        <item xml:id="q10" n="9">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="83r" to="92v"/>                                            
+                                        </item>
+                                        <item xml:id="q11" n="10">
+                                            <dim unit="leaf">12</dim>
+                                            <locus from="93r" to="104v"/>                                            
+                                        </item>
+                                        <item xml:id="q12" n="11">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="105r" to="112v"/>                                            
+                                        </item>
+                                        <item xml:id="q13" n="12">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="113r" to="122v"/>                                            
+                                        </item>
+                                        <item xml:id="q14" n="13">
+                                            <dim unit="leaf">4</dim>
+                                            <locus from="123r" to="126v"/>                                            
+                                        </item>
+                                    </list>
+                                </collation>
+                                <condition key="good"/>
+                            </supportDesc>
+                            
+                            <layoutDesc>
+                                <layout columns="2" writtenLines="18 19">                          
+                            <ab type="pricking">Pricking is visible.</ab>
+                            <ab type="pricking">Primary pricks are visible, horizontal pricks are visible.</ab>
+                            <ab type="ruling">Ruling is visible.</ab>
+                                    <ab type="ruling">The upper and lower lines are written above the ruled line. </ab>
+                            <locus from="3r" to="126v"></locus>
+                            </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                       
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
```

Comment:
I think if you say that there are two hands, you need to have two descriptions of hands and then explain who wrote what.